### DISCUSSION TITLE
Scenario starters

### DISCUSSION THREAD

**Discussion by jmankoff** [make4all/psst#96] [Ideas]
Here are some things to try to support easily (maybe a test for our tool AND the user interface)
 
[Nametag](https://makecode.microbit.org/projects/name-tag); [Flashing heart](https://makecode.microbit.org/projects/flashing-heart): State of LEDs in 2D over time
[Smiley Buttons](https://makecode.microbit.org/projects/smiley-buttons): Same PLUS state of buttons
[Dice](https://makecode.microbit.org/projects/dice): Same PLUS accelerometer; 
[Chat](https://makecode.microbit.org/projects/micro-chat): Wireless communication status
[Indoor/outdoor thermo](https://microbit.org/projects/make-it-code-it/indoor-outdoor-thermometer/);  [Thermometer](https://microbit.org/projects/make-it-code-it/thermometer/); LED; thermometer; button
[Tilt Alarm](https://microbit.org/projects/make-it-code-it/simple-tilt-alarm/): Audio; accelerometer
[Touch timer](https://microbit.org/projects/make-it-code-it/touch-timer/): LEDs; Capacitive sensing
[Clapping timer](https://microbit.org/projects/make-it-code-it/clapometer/): Volume sensing
[Sound compass](https://microbit.org/projects/make-it-code-it/sound-compass/): Plays a sound when pointed north
[Light alarm](https://microbit.org/projects/make-it-code-it/light-alarm/): Plays a sound when light is above a certain level
[Wireless data logger](https://microbit.org/projects/make-it-code-it/makecode-wireless-data-logger/)

**Comment by jmankoff** [comment]
@pelikhan @jamesadevine @jrthompson33 @venkateshpotluri  just tagging you for this list of ideas as well.

**Reply by pelikhan** [reply]
We do have a plant activity where we measure soil moisture and light level. It's a fairly slow data stream.
